layersvt/device_simulation_examples/sdk_sample_configs/minimum_vulkan_1_2.json

```diff
@@ -0,0 +1,1133 @@
+        "storageBuffer8BitAccess": 0,
+        "uniformAndStorageBuffer8BitAccess": 0,
+        "storagePushConstant8": 0
+    },
+    "VkPhysicalDeviceDescriptorIndexingFeatures": {
+        "shaderInputAttachmentArrayDynamicIndexing": 0,
+        "shaderUniformTexelBufferArrayDynamicIndexing": 0,
+        "shaderStorageTexelBufferArrayDynamicIndexing": 0,
+        "shaderUniformBufferArrayNonUniformIndexing": 0,
+        "shaderSampledImageArrayNonUniformIndexing": 0,
+        "shaderStorageBufferArrayNonUniformIndexing": 0,
+        "shaderStorageImageArrayNonUniformIndexing": 0,
+        "shaderInputAttachmentArrayNonUniformIndexing": 0,
+        "shaderUniformTexelBufferArrayNonUniformIndexing": 0,
+        "shaderStorageTexelBufferArrayNonUniformIndexing": 0,
+        "descriptorBindingUniformBufferUpdateAfterBind": 0,
+        "descriptorBindingSampledImageUpdateAfterBind": 0,
+        "descriptorBindingStorageImageUpdateAfterBind": 0,
+        "descriptorBindingStorageBufferUpdateAfterBind": 0,
+        "descriptorBindingUniformTexelBufferUpdateAfterBind": 0,
+        "descriptorBindingStorageTexelBufferUpdateAfterBind": 0,
+        "descriptorBindingUpdateUnusedWhilePending": 0,
+        "descriptorBindingPartiallyBound": 0,
+        "descriptorBindingVariableDescriptorCount": 0,
+        "runtimeDescriptorArray": 0
+    },
+    "VkPhysicalDeviceHostQueryResetFeatures": {
+        "hostQueryReset": 0
+    },
+    "VkPhysicalDeviceImagelessFramebufferFeatures": {
+        "imagelessFramebuffer": 0
+    },
+    "VkPhysicalDeviceMultiviewFeatures": {
+        "multiview": 1,
+        "multiviewGeometryShader": 0,
+        "multiviewTessellationShader": 0
+    },
+    "VkPhysicalDeviceProtectedMemoryFeatures": {
+        "protectedMemory": 0
+    },
+    "VkPhysicalDeviceSamplerYcbcrConversionFeatures": {
+        "samplerYcbcrConversion": 0
+    },
+    "VkPhysicalDeviceScalarBlockLayoutFeatures": {
+        "scalarBlockLayout": 0
+    },
+    "VkPhysicalDeviceSeparateDepthStencilLayoutsFeatures": {
+        "separateDepthStencilLayouts": 0
+    },
+    "VkPhysicalDeviceShaderAtomicInt64Features": {
+        "shaderBufferInt64Atomics": 1,
+        "shaderSharedInt64Atomics": 1
+    },
+    "VkPhysicalDeviceShaderDrawParametersFeatures": {
+        "shaderDrawParameters": 0
+    },
+    "VkPhysicalDeviceShaderFloat16Int8Features": {
+        "shaderFloat16": 0,
+        "shaderInt8": 0
+    },
+    "VkPhysicalDeviceShaderSubgroupExtendedTypesFeatures": {
+        "shaderSubgroupExtendedTypes": 0
+    },
+    "VkPhysicalDeviceTimelineSemaphoreFeatures": {
+        "timelineSemaphore": 0
+    },
+    "VkPhysicalDeviceUniformBufferStandardLayoutFeatures": {
+        "uniformBufferStandardLayout": 1
+    },
+    "VkPhysicalDeviceVariablePointersFeatures": {
+        "variablePointersStorageBuffer": 0,
+        "variablePointers": 0
+    },
+    "VkPhysicalDeviceVulkanMemoryModelFeatures": {
+        "vulkanMemoryModel": 1,
```

Comment:
This is not supported everywhere:
https://vulkan.gpuinfo.org/listdevicescoverage.php?core=1.2&feature=vulkanMemoryModel&platform=windows&option=not